src/WebJobs.Script/Config/IFunctionsHostingConfiguration.cs

```diff
@@ -3,8 +3,10 @@
 
 namespace Microsoft.Azure.WebJobs.Script.Config
 {
-    internal interface IFunctionsHostingConfiguration
+    public interface IFunctionsHostingConfiguration
     {
         bool FunctionsWorkerDynamicConcurrencyEnabled { get; }
+
+        string GetValue(string key, string defaultValue = null);
```

Comment:
Previously this interface provided a programming model on top of raw config. Why do we need to expose the raw config now? I.e. why not just expose nice properties for the values you want to expose (e.g. ScaleControllerFeatureFlags), or perhaps expose booleans for the flags we care about?

Reply:
For TBS we need just read a value by a key. Format:
"ENABLE_FEATUREX=1,A=B,TimeOut=123,eventhubtargetscaler=1,servicebustargetscaler=1,queuetargetscaler=1"

Also the method will be useful  to enable other features using `FunctionsHostingConfiguration`.